src/Codeception/Command/Completion.php

```diff
@@ -0,0 +1,79 @@
+<?php
+namespace Codeception\Command;
+
+use Codeception\Configuration;
+use Stecman\Component\Symfony\Console\BashCompletion\Completion as ConsoleCompletion;
+use Stecman\Component\Symfony\Console\BashCompletion\CompletionCommand;
+use Stecman\Component\Symfony\Console\BashCompletion\CompletionHandler;
+use Stecman\Component\Symfony\Console\BashCompletion\Completion\ShellPathCompletion as ShellPathCompletion;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class Completion extends CompletionCommand
+{
+    protected function configureCompletion(CompletionHandler $handler)
+    {
+        // Can't set for all commands, because it wouldn't work well with generate:suite
+        $suiteCommands = [
+            'run',
+            'config:validate',
+            'console',
+            'dry-run',
+            'generate:cept',
+            'generate:cest',
+            'generate:feature',
+            'generate:phpunit',
+            'generate:scenarios',
+            'generate:stepobject',
+            'generate:test',
+            'gherkin:snippets',
+            'gherkin:steps'
+        ];
+
+        foreach ($suiteCommands as $suiteCommand) {
+            $handler->addHandler(new ConsoleCompletion(
+                $suiteCommand,
+                'suite',
+                ConsoleCompletion::TYPE_ARGUMENT,
+                Configuration::suites()
+            ));
+        }
+
+        $handler->addHandlers([
+            new ShellPathCompletion(
+                ConsoleCompletion::ALL_COMMANDS,
+                'path',
+                ConsoleCompletion::TYPE_ARGUMENT
+            ),
+            new ShellPathCompletion(
+                ConsoleCompletion::ALL_COMMANDS,
+                'test',
+                ConsoleCompletion::TYPE_ARGUMENT
+            ),
+        ]);
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        if ($input->getOption('generate-hook') && $input->getOption('use-vendor-bin')) {
+            global $argv;
+            $argv[0] = 'vendor/bin/' . basename($argv[0]);
```

Comment:
I think that `use-vendor-bin` option is unnecessary and confusing (and actually broken, because `$argv[0]` is not used if `program` option is given),  using `--program vendor/bin/codecept` gives the same result.
Please document effects of using different values of `program` option.

Also I think that it would be good to suggest to use  ```eval `codecept _completion --generate-hook --program codecept` ``` for quick evaluation of code completion without modifying rc files.